src/NotesLoader.cpp

```diff
@@ -13,7 +13,7 @@
 
 void NotesLoader::GetMainAndSubTitlesFromFullTitle( const RString &sFullTitle, RString &sMainTitleOut, RString &sSubTitleOut )
 {
-	static const std::string_view sLeftSeps[] = { "\t", " -", " ~", " (", " [" };
+	static const std::string sLeftSeps[] = { "\t", " -", " ~", " (", " [" };
```

Comment:
adding a std::string_view constructor to RString complains that std::string_view isn't available in some files, and indeed some files, e.g. LoadingWindow_Gtk.cpp, are using `-std=gnu++11`

Reply:
ah wait, easy CMake fix